uitest/README.md

```diff
@@ -0,0 +1,44 @@
+## Dev Home UI Testing
+### How to run UI tests?
+1. From the command prompt, start the Windows Application Driver service:
+```cmd
+"C:\Program Files\Windows Application Driver\WinAppDriver.exe"
+```
+2. From Visual Studio, navigate to the "Test Explorer" tab and locate the `DevHome.UITest` set of tests.
+3. Select a test and run it.
+    - Note: Once the test starts, avoid interacting with your machine (e.g. move mouse, use keyboard) to allow the test to navigate the Dev Home app and execute the test.
+
+### How to configure the application settings for a Dev Home release?
```

Comment:
I would put this at the end, after how to run with a different build. We probably won't be modifying these files as often, and I think it'd be easier to have all the common cases at the top and details later

Reply:
`appsettings.json` is more likely to be updated compared to the `test.runsettings`.

The test runsettings is linked inside the `csproj`, so as a UI tester, the file should not be modified unless locally we temporarily need to tell the UI test to run same as the pipeline (`prod` or `canary`) without committing this change to the repository. On the other hand, app settings are used for configuring the UI test methods (more details here https://github.com/microsoft/devhome/pull/1313#discussion_r1270906474).

Reply:
I get the different use cases, that's why I think `test.runsettings` is more likely to need to be changed (without committing) than `appsettings.json`. I think adding new options that we want to be able to configure would be something rare, while running the tests against `prod` or `canary` could easily be part of adding tests (checking whether the current releases passes it, or ensuring it fails to contrast with the dev build passing)

Reply:
Done